Plugins~/Src/MeshSyncClientBlender/python/3.0.1/unity_mesh_sync.py

```diff
@@ -113,15 +114,26 @@ class MESHSYNC_OT_AutoSync(bpy.types.Operator):
     def __del__(self):
         MESHSYNC_OT_AutoSync._timer = None
 
+    def execute(self, context):
+        return self.invoke(context, None)
+
     def invoke(self, context, event):
         scene = bpy.context.scene
         if not MESHSYNC_OT_AutoSync._timer:
             scene.meshsync_auto_sync = True
             if not scene.meshsync_auto_sync:
                 # server not available
                 return {'FINISHED'}
-            MESHSYNC_OT_AutoSync._timer = context.window_manager.event_timer_add(1.0 / 3.0, window=context.window)
+            update_step = 0.0001 # 1.0/3.0
```

Comment:
Auto sync executes a diff check on every scene object. For geometry nodes, this is probably fine but I wonder how it will perform for projects with large scenes or a lot of geometry